library/objective-c/EnvoyEngine.h

```diff
@@ -394,12 +399,28 @@ extern const int kEnvoyFailure;
 
 @end
 
+#pragma mark - EnvoyLogger
+
+// Logging interface.
+@interface EnvoyLogger : NSObject
+
+@property (nonatomic, copy, nullable) void (^log)(NSString *);
```

Comment:
Would it make sense to have the `EnvoyLogger` itself be `nil` instead of having an empty object with a `nil` closure?